src/HotChocolate/Data/src/NHibernate/Extensions/NHibernateFactory.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="NHibernateFactory.cs" company="GEODIS">
+// Copyright (c) 2021 GEODIS.
+// All rights reserved. www.geodis.com
+// Reproduction or transmission in whole or in part, in any form or by
+//  any means, electronic, mechanical or otherwise, is prohibited without the
+//  prior written consent of the copyright owner.
+// </copyright>
+// 
+// <application>All</application>
+// <module>HotChoclate.Data.NHibernate</module>
+// <author>Viswanathan, Satish</author>
+// <createddate>2021-09-02</createddate>
+// <lastchangedby>Viswanathan, Satish</lastchangedby>
+// <lastchangeddate>2021-09-02</lastchangeddate>
```

Comment:
We can only accept code that does not have copyright. I guess this is just a standard template where you did forget to clean.

```suggestion
```

Reply:
My bad. I've removed those.